README.md

```diff
@@ -6,29 +6,26 @@ It contains all the necessary configuration and some placeholder files to get yo
 The best way to create your own project based on this starter is [start.vaadin.com](https://start.vaadin.com/) - you can get only the necessary parts and choose the package naming you want to use.
 
 ## Running the Application
-There are two ways to run the application :  using `mvn spring-boot:run` or by running the `Application` class directly from your IDE.
 
-You can use any IDE of your preference,but we suggest Eclipse or Intellij IDEA.
-Below are the configuration details to start the project using a `spring-boot:run` command. Both Eclipse and Intellij IDEA are covered.
+While developing the application you should always launch the `Application` class in debug mode from your IDE. 
```

Comment:
This statement seems a bit too opinionated to me.
I would prefer something like `While developing the application, it is recommended to launch the `Application` class in debug mode from your IDE.` 
But it is just an opinion, feel free to discard the suggestion.

Reply:
Updated